tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -499,6 +503,16 @@ def add(self, ticket: SessionTicket) -> None:
     def pop(self, label: bytes) -> Optional[SessionTicket]:
         return self.tickets.pop(label, None)
 
+class WebTransportCertificateGeneration(Enum):
```

Comment:
If we're assigning them integer values, we can use IntEnum and skip the additional import, right?

```suggestion
class WebTransportCertificateGeneration(IntEnum):
```